src/ui/components/Hero/styles.scss

```diff
@@ -15,54 +15,3 @@
     padding: $padding-page-l;
   }
 }
-
-.Hero-contents {
```

Comment:
this is all very specific to the current HomeHeroBanner, so I moved a lot of these styles over to `HomeHeroBanner.scss`

Reply:
okay, I guess that's fine